smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/commit/KafkaCheckpointCommit.java

```diff
@@ -398,7 +398,7 @@ public LastStateStoredTooLongAgoException(TopicPartition topic, long time, long
         }
     }
 
-    static class CheckpointState<T> {
+    public static class CheckpointState<T> {
```

Comment:
That seems to be important (while unrelated to that PR)

Reply:
This is why the revapi was complaining. But I didn't see that before